src/drivers/clock/integratorcp_clock.c

```diff
@@ -45,6 +47,15 @@
 EMBOX_UNIT_INIT(integratorcp_init);
 
 static int integratorcp_clock_setup(struct time_dev_conf * conf) {
+	if (NULL == mmap_device_memory(
+		(void*) TIMER_BASE,
+		0x28,
```

Comment:
Why region size initialization is different for integrator_clock & cortexa9_timer